scripts/conda/build.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+set -ex
+[ -z "$PREFIX" ] && export PREFIX=$CONDA_PREFIX
+
+# Make sure -fPIC is not in CXXFLAGS (that some conda packages may
+# add):
+export CXXFLAGS="`echo $CXXFLAGS | sed 's/-fPIC//'`"
+
+# go overwrites CC and CXX with nonsense (see
+# https://github.com/conda-forge/go-feedstock/issues/47), hence we
+# redefine these below. Reset GO env variables for omniscidb build
+# (IIRC, it is needed for CUDA support):
+export CGO_ENABLED=1
+export CGO_LDFLAGS=
+export CGO_CFLAGS=$CFLAGS
+export CGO_CPPFLAGS=
+
```

Comment:
https://github.com/conda-forge/go-feedstock/issues/47 is resolved, so we should be able to delete the above.